content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Upcoming ROS 1 End of Life"
+author:  Kat Scott 
+subtitle: Migration Paths for ROS 1 Users
+created_at: 2024-12-16 16:21:05 -0700
+kind: article
+header_class:  bg-photo 
+header_image:  /imgs/eol-bg.png 
+---
+
+## ROS 1 Noetic Ninjemys goes End-of-Life 2025-05-31
+
+When we [announced](https://www.openrobotics.org/blog/2020/5/23/noetic-ninjemys-the-last-official-ros-1-release) ROS 1 Noetic Ninjemys on May 23rd, 2020 we made it very clear that it would be the last official ROS 1 distribution. 
+In the proceeding five years we focused our efforts on making ROS 2 a world class framework for professional robot development, and our user community seems to agree.
+According to September 2024 download data from our servers ROS 2 downloads now make up almost 80% of all ROS downloads.
+We've now seen ROS 2 used successfully in production by a large number of companies, across dozens of industries, and we think it is time for the last remaining ROS users to make the switch. 
```

Comment:
I'd recommend putting this info into a section about why people should migrate to ROS 2